MovieDB/src/Main.java

```diff
@@ -0,0 +1,51 @@
+import content.Movie;
+import content.MovieList;
+import content.Show;
+import services.Service;
+import users.Admin;
+import users.Client;
+import users.User;
+
+import java.util.*;
+
+
+public class Main {
+    public static void main(String[] args){
```

Comment:
[Fix] daca nu ai facut meniu interactiv, si ai ales varianta de "demo", atunci include niste mesaje ajutatoare, sa inteleaga lumea ce sunt toate chestiile alea printate in consola
![image](https://user-images.githubusercontent.com/67580205/161397222-5ba8f133-3125-4560-9dd0-57f50d70528d.png)
asa, nu au nicio noima